ksql-rest-app/src/test/resources/ksql-plan-schema/schema.json

```
@@ -288,24 +286,18 @@
       "title" : "streamAggregateV1",
       "required" : [ "@type", "properties", "source", "formats", "nonFuncColumnCount", "aggregations" ]
     },
-    "DefaultExecutionStepProperties" : {
+    "ExecutionStepProperties" : {
```

We should plan for extension where new steps are added that need a different set of properties, i.e. we might want this to be versioned... e.g. 

```suggestion
    "ExecutionStepPropertiesV1" : {
```